pkg/cloudcore/apis/config/types.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import commonconfig "github.com/kubeedge/kubeedge/pkg/common/apis/config"
+
+type CloudCoreConfig struct {
+	Kube             *KubeConfig             `json:"kube,omitempty"`
```

Comment:
I guess we need yaml definitions as well ?

Reply:
we do not need yaml definitions , Keep the same with k8s。 we can use `sigs.k8s.io/yaml`  library to  convert.